### ISSUE TITLE
Indicate to user when making a query via WPS that system/app is processing

### ISSUE THREAD

**Issue by AnaBelgun** [TerriaJS/terriajs#3087]
Geoglam user feedback: 
When making queries using WPS, due to the size of the polygons / data, the response takes a while and there is no indication to the user that nothing is wrong and it will be eventually completed. 

One suggestion was to replace "loading" with " processing"

![image](https://user-images.githubusercontent.com/18452161/45795740-84b5c000-bce0-11e8-8d55-eb252d36fe39.png)

http://map.geo-rapp.org/#share=s-A3RNNto4fioCpM5xT0mtxZF4PkL

**Comment by chloeleichen** [comment]
@philipgrimmett you happy with the solution (change 'loading' into 'processing')?

**Comment by philipgrimmett** [comment]
@chloeleichen yep, "processing" is probably more appropriate in this instance. Not sure it'll have the desired effect, though. Can we be more specific about what is being processed? Do we have any way to track progress?
